src/usr/local/www/services_dyndns_edit.php

```diff
@@ -599,22 +584,6 @@ function setVisible(service) {
 				hideInput('zoneid', true);
 				hideInput('ttl', false);
 				break;
-			case "domeneshop":
-		        case "domeneshop-v6":
-				hideGroupInput('domainname', true);
-				hideInput('resultmatch', true);
-				hideInput('updateurl', true);
-				hideInput('requestif', true);
-				hideCheckbox('curl_ipresolve_v4', true);
-				hideCheckbox('curl_ssl_verifypeer', true);
-				hideInput('username', false);
```

Comment:
This line for `username` is here for `domenshop` but it's not in your combined version below.

Reply:
I was thinking about that. If I get the logic correctly, then I think there is a bug. There is only one provider, which hides the `username` and none of the providers shows it again, so it should stay hidden. I should test that hypothesis.

If that is the case, then I think the code should hide all fields in one place and then show those that are specific for the provider. Or at least have some "defaults" hidden/shown and only difference per provider. I'll look into this a bit soon.

Any case, I revert merging from this PR and do that separately.